src/calibration.rs

```diff
@@ -0,0 +1,88 @@
+//! Factory-programmed calibration data
+//!
+//! The STM32L0 contains a few read-only registers with factory calibration
+//! data that are written during production.
+
+macro_rules! define_ptr_type {
+    ($name: ident, $ptr: expr) => {
+        impl $name {
+            fn ptr() -> *const Self {
+                $ptr as *const _
+            }
+
+            /// Returns a wrapped reference to the value in flash memory
+            pub fn get() -> &'static Self {
+                unsafe { &*Self::ptr() }
+            }
+        }
+    };
+}
+
+/// Internal voltage reference (V_REFINT)
+///
+/// The internal voltage reference provides a stable (bandgap) voltage output
+/// for the ADC and Comparators. The precise voltage of V_REFINT is
+/// individually measured for each part by ST during production test and stored
+/// in the system memory area. It is accessible in read-only mode.
```

Comment:
What's "read-only mode"? Does that mean the register can only be read, not written, or is there some mode switching involved that I'm not aware of?

Reply:
The "read-only mode" is actually copy paste from the datasheet, but I agree that the wording is confusing (as are ST datasheets in general :wink:)

Reply:
I don't find it confusing (it is written by ST in the factory, but accessible to the developer in read-only mode) but I can also just remove that part of the sentence, since it doesn't really add anything useful.